src/app/Dashboard/dashboard-utils.tsx

```diff
@@ -418,3 +418,27 @@ export interface DashboardCardTypeProps {
   isFullHeight?: boolean;
   actions?: JSX.Element[];
 }
+
+/* CARD COMPONENTS */
+
+export const transformAADescription = (description: string): JSX.Element => {
+  const splitDesc = description.split('\n\n');
+  const boldRegex = /^([^:]+:\s?)/; // match text up to and including the first colon
```

Comment:
Lets try this:

```ts
^([^:]+):\s*(.*)
```

Tester: https://regex101.com/r/iP2RoX/1

This way, u can pull the matched group out easily.

```
// Must check matched first
const [_, boldText, content] = matched;
```

Reply:
also, we need to check the content (hide it maybe?) as it might be empty (not sure when but best to check).

Reply:
Opps nvmind :)) I guess this could be a nice refactoring idea then :)